community-files/README.md

```diff
@@ -0,0 +1,3 @@
+# community-files
+
+Contains required community files for all repositories.
```

Comment:
I'm wondering if we should add little blurbs in the Readme.md on each individual file. That way, we could include specifics on how we are managing the merging of the said file into other repositories for people to easily access it. What do you think? Here is an example of what that could look like, but you can mess with the format of it.

```suggestion
Contains required community files for all repositories.

---

LICENSE
- purpose of the file
- How the file is merged in the parent directories

---

CODE_OF_CONDUCT.md
- Contains the code of conduct describing the standards and responsibilities to be upheld by those who contribute to the Symbiflow projects
- Merging to parent repositories is not yet implemented
```

Reply:
I'm not sure I understand what you mean by how the file is merged with parent directories. Essentially all of the files in this repo need to be at the top level in order to function properly, so it would probably just be one script that moves them all out of folders up one directory. If all of the files in each folder have the same general purpose and same general merge strategy, I think it might be redundant to put additional lines for each file.

Reply:
I meant parent repositories. But you are right that they will be the same for each file, so that is unnecessary. Good point.

Reply:
I'll go ahead with the merge